src/test/ui/target-feature/missing-plusminus.stderr

```diff
@@ -0,0 +1,6 @@
+warning: unknown feature specified for `-Ctarget-feature`: `banana`
+   |
+   = note: features must begin with a `+` to enable or `-` to disable it
```

Comment:
Should we also have a test for `-Ctarget-feature=rdrand` where we tell them "you have to either enable it or disable it"?

Reply:
Done!